goja/goja_manager.py

```
@@ -162,25 +162,32 @@ def verify_goja_output(gate_path, goja_path):
 
     else:
       path_coincidences = args.path_goja_output + args.simulation_name + "_COINCIDENCES"
-      if os.path.isfile(path_coincidences):
+      try:
         os.unlink(path_coincidences)
+      except:
+        pass
```

This is fine for quick test script. Silently catching all exceptions will cause a lot of trouble. Do something like this:
```
        try:
            os.mkdir(run_dir)
        except OSError as e:
            if e.errno == errno.EEXIST:
                print "Fit %s already exists - skipping." % run_dir
                continue
            else:
                raise e
```

- Replace errno.EEXIST with errno.ENOENT - https://docs.python.org/2/library/errno.html#errno.ENOENT
- Replace continue with pass
- Remove the print statement

---

Done